web3rpc/rpc-specs/code-samples/curl/klay/transaction/sendRawTransaction.sh

```diff
@@ -6,5 +6,5 @@ curl -X 'POST' \
   "method": "klay_sendRawTransaction",
   "id": 1,
   "jsonrpc": "2.0",
-  "params": ["0xf86680850ba43b7400829999948c9f4468ae04fb3d79c80f6eacf0e4e1dd21deee01808207f4a092d389e81c665711cb71cb6decc33fda5f990b4a62a309a7f5ae704a24a9848fa0458e5e3ceceb05371241be9424ac8062fcefee8373ffdacf2935860e5cc7d9b1"]
+  "params": [["0x01f8ae8203e98085066720b300830186a094cb00ba2cab67a3771f9ca1fa48fda8881b45775080b844095ea7b30000000000000000000000004b41783732810b731569e4d944f59372f411bea2ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc080a0a288a06b1f6dc556e9e651d30cc8b985e58ecb418facb9666d5c49af5f4331f7a04068642611bf8f99d31e61e4386cc91130d3b55c386ec97b02a9c98d01aa61af", "0xf9010c0585066720b3008307a120944b41783732810b731569e4d944f59372f411bea280b8a480426901000000000000000000000000cb00ba2cab67a3771f9ca1fa48fda8881b457750000000000000000000000000000000000000000000000000494a02fda18ed1ce0000000000000000000000000000000000000000000000004596784b50f8f8000000000000000000000000000000000000000000000000000032e6c90c04f80000000000000000000000000000000000000000000000000000000000683570348207f5a01ea30dd90c2b2938c664236c910b0ee41e9fa949c4917e726b91a1c543095304a07afd83df326fbe3f0a305433139165d5b3872378e1a9129bc3ab3ab68da186a2"]]
```

Comment:
This one is not two-level array.

kaia_sendRawTransaction and
kaia_sendRawTransactions are different.

Reply:
@Sotatek-LongPham2 Please revert this file to before PR. The `kaia_sendRawTransaction` API receives ONE raw transaction string.

You can know simply by executing this file
```
./sendRawTransaction.sh
{"jsonrpc":"2.0","id":1,"error":{"code":-32602,"message":"too many arguments, want at most 1"}}
```